ensure_flutter_version.go

```diff
@@ -0,0 +1,418 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/bitrise-io/go-flutter/flutterproject"
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/command/git"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
```

Comment:
When writing new code, we try to use the `/v2` variant of these utils (which equals to the main branch of the repo). And only use these packages (the `v1` git branch) when a given utility is not migrated to the v2 package yet.